.github/workflows/trigger-automation-from-comment.yml

```
@@ -0,0 +1,263 @@
+    types: [created]
+
+jobs:
+  parse-and-trigger:
+    # Only run on PR comments (not regular issues)
+    if: github.event.issue.pull_request && contains(github.event.comment.body, 'run test automation')
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check comment format and extract parameters
+        id: parse_comment
+        run: |
```

actionlint (1.7.7)

"github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)